### ISSUE TITLE
В метрику отправляется некорректное количество пользователей

### ISSUE THREAD

**Issue by AlexGerm** [yandexmobile/metrica-sdk-ios#72]
В `AppDelegate` в методе `didFinishLaunchingWithOptions` делаю активацию метрики:
`YMMYandexMetrica.activate(with: YMMYandexMetricaConfiguration(apiKey: token))`, затем мне нужно в этом же методе отправить в метрику некоторые события, связанные с запуском приложения. Делаю это через `YMMYandexMetrica.reportEvent`. Количество событий и устройств, которые приходят в метрику - корректно. Количество пользователей всегда остаётся равным 0 вне зависимости от того устанавливается ли с нуля приложение или перезапускается.

**Comment by AlexGerm** [comment]
Спасибо за разъяснения. Думаю, было бы неплохо эту инфу добавить в документацию.

**Comment by AlexGerm** [comment]
Уточните, пожалуйста, ещё такой момент. Сейчас наблюдаю такую ситуацию, что счётчик пользователей не увеличивается, если событие с одним и тем же deviceID ранее было отправлено в состоянии inactive, а потом в active. Это корректное поведение? 
Есть ли какой-то способ принудительно увеличивать количество пользователей в этом случае?

**Comment by bamx23** [comment]
Так происходит для всех событий после выхода в Active?
В принципе такого быть не должно. При переходе в Active создается сессия, все попадающие туда события считаются "с пользователями", не зависимо от того, были ли они отправлены ранее. Может быть SDK не успело получить нотификацию от NotificationCenter'а?

**Comment by bamx23** [comment]
Да, и лучше делать это после активации АппМетрики. При этом нет гарантий, что система будет вызывать подписчиков в том же порядке.
